frontend/app/src/utils/collection.ts

```diff
@@ -86,14 +86,19 @@ export const setupEntryLimit = (
   };
 };
 
-export const defaultOptions = <T extends Object>(
-  orderBy?: keyof T
-): TablePagination<T> => {
+export const defaultOptions = <T extends Object>(defaultSortBy?: {
```

Comment:
if this doesn't affect other use-cases of this function, then, it's all good.

Reply:
Sure, I believe it doesn't affect other use-cases, 
also it's only used on 2 occasions inside file `filter-paginate`